resources/io/github/abogoyavlensky/clojure_stack_lite/root/Dockerfile

```diff
@@ -1,18 +1,28 @@
 FROM clojure:temurin-21-tools-deps-1.12.0.1479-alpine AS build
 
-# Install bb
-RUN apk --no-cache add curl \
-  && curl -sLO https://raw.githubusercontent.com/babashka/babashka/master/install \
-  && chmod +x install \
-  && ./install --version 1.12.197 --static
+WORKDIR /app
+
+ARG BB_VERSION=1.12.197
+ARG TAILWIND_VERSION=v4.1.3
 
 # Install tailwindcss
-RUN curl -sL -o /usr/local/bin/tailwindcss https://github.com/tailwindlabs/tailwindcss/releases/download/v4.1.3/tailwindcss-linux-x64-musl \
-  && chmod +x /usr/local/bin/tailwindcss
+RUN wget -qO /usr/local/bin/tailwindcss "https://github.com/tailwindlabs/tailwindcss/releases/download/${TAILWIND_VERSION}/tailwindcss-linux-x64-musl" \
+    && chmod +x /usr/local/bin/tailwindcss
+
+# Install bb
+RUN wget -qO- https://raw.githubusercontent.com/babashka/babashka/master/install \
+    | bash -s -- --version "${BB_VERSION}" --static
+
+# bb: (trigger) install of Clojure tools and download deps
+COPY bb.edn vendor ./
+RUN bb prepare
+
+# clj: download deps
+COPY deps.edn vendor ./
+RUN clojure -X:build:deps prep
 
 # Build uberjar
-WORKDIR /app
-COPY . /app
+COPY . .
```

Comment:
tbh, I prefer using an explicit full path: `COPY . /app`. This way, I don't have to search for the workdir definition throughout the entire file.

Reply:
Restored it.